Cue/app/tabs/library/TagsTableRow.js

```diff
@@ -47,8 +47,9 @@ export default class TagsTableRow extends React.Component {
   _onEndEditing = ({nativeEvent: {text}}) => {
     this.textInputRef.clear()
 
-    let tag = text.trim().toLowerCase()
-      .replace(new RegExp('[^-a-z ]', 'g'), '')
+    let tag = text
+      .trim()
+      .toLowerCase()
```

Comment:
Indeed. I didn't see any adverse effects trying it in the simulator with Korean or other Japanese characters either.

<img width="192" alt="screen shot 2017-03-28 at 9 17 23 pm" src="https://cloud.githubusercontent.com/assets/13400887/24434181/ffef4c44-13fb-11e7-9fc3-bbeda4d125be.png">